osgp/platform/osgp-adapter-kafka-distributionautomation/src/main/java/org/opensmartgridplatform/adapter/kafka/da/application/mapping/ScadaMeasurementPublishedEventConverter.java

```diff
@@ -41,7 +37,7 @@
  * current_returned_L2; current_returned_L3;
  * <p>
  * ls peak shaving measurement: ean_code + the values of
- * LsPeakShavingMeasurementType seperated by semicolons.
+ * LsPeakShavingMeasurementType separated by semicolons.
  */
 public class ScadaMeasurementPublishedEventConverter extends CustomConverter<String, ScadaMeasurementPublishedEvent> {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS2160)